docs/next/guides/internationalization/layout-direction.md

```diff
@@ -0,0 +1,220 @@
+
+## About layout direction
+
+To properly display Handsontable's UI and data in LTR languages (such as English, Chinese, or Russian) and RTL languages (such as Arabic, Persian, or Hebrew), 
+you can configure your grid's layout direction.
+
+We advise to use the RTL direction with a right-to-left language of the user interface. You can use a built-in Arabic [language](@/guides/internationalization/language.md) translation, which is right-to-left, or make a custom translation to include any other language.
+
+By default, Handsontable's layout direction is set automatically, based on on the value of your HTML document's [`dir`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/dir) attribute.
+
+You can:
+- [Set the layout direction automatically](#setting-the-layout-direction-automatically)
+- [Set the layout direction to RTL](#setting-the-layout-direction-to-rtl)
+- [Set the layout direction to LTR](#setting-the-layout-direction-to-ltr)
+
+### RTL demo
+
+To try out Handsontable's RTL support, check out the demo below:
+
+::: example #example1 :hot-lang
+```js
+const container = document.querySelector('#example1');
+
+const hot = new Handsontable(container, {
+  licenseKey: 'non-commercial-and-evaluation',
+  data: [
+  ['', 'Tesla', 'Volvo', 'Toyota', 'Ford'],
+  ['2019', 10, 11, 12, 13],
+  ['2020', 20, 11, 14, 13],
+  ['2021', 30, 15, 12, 13]
+  ],
+  colHeaders: true,
+  rowHeaders: true,
+  // render Handsontable from the right to the left
+  layoutDirection: 'rtl',
+  // load a RTL language
+  language: 'ar-AR',
+  // enable some options that exemplify the layout direction
+  dropdownMenu: true,
+  filters: true,
+  contextMenu: true
+});
+```
+:::
+
+### Elements affected by layout direction
+
+Setting a different layout direction affects the behavior of the following areas of Handsontable:
+
+| Element                                                                                           | Behavior in the LTR layout direction                                                                                                                             | Behavior in the RTL layout direction
+| ------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------
+| Starting and ending edges                                                                         | The left-hand edge is considered the starting edge of the grid, and the right-hand edge is considered the ending edge of the grid.                               | The left-hand edge is considered the starting edge of the grid, and the right-hand edge is considered the ending edge of the grid.
```

Comment:
```suggestion
| Starting and ending edges                                                                         | The left-hand edge is considered the starting edge of the grid, and the right-hand edge is considered the ending edge of the grid.                               | The left-hand edge is considered the ending edge of the grid, and the right-hand edge is considered the starting edge of the grid.
```

Reply:
Thanks! I don't know how I could miss that. I had a different explanation in mind, now committed at 2982639278ad159bbd78a01129368a8475aca936